en/donate.html

```diff
@@ -44,7 +44,38 @@
         </div>
     </nav>
     
-    <main class="bg-gray-100"></main>
+    <main class="bg-gray-100">
+
+        <div class="grid items-center md:grid-cols-[repeat(2,_1fr)] w-full relative mb-0">
+            <img class="shadow-2xl rounded-tr-md" src="../assets/katt-yukawa-K0E6E0a0R3A-unsplash 1.png" alt="An assortment of vegetables">
```

Comment:
Same comment about image filenames (doesn't have to be done now, we can rename these after merging to make things simpler)